docs/docs/sign-in-out.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Sign In and Out
+layout: docs
+---
+
+To sign in and out of Tableau Server, call the `Auth.sign_in` and `Auth.sign_out` functions like so:
+
+```py
+import tableauserverclient as TSC
+
+tableau_auth = TSC.TableauAuth('USERNAME', 'PASSWORD')
+server = TSC.Server('http://SERVER_URL')
+
+server.auth.sign_in(tableau_auth)
+
+# Do awesome things here!
+
+server.auth.sign_out()
+```
+
+This code does the following things:
+
+* Creates an authentication object that stores your user name and password.
```

Comment:
Honest question: Does anyone care that it does all these things?

sign_in signs you in, sign_out signs you out.

You might want to note that it's a normal session, so it's got a time limit (2 hours, 4 hours, I forget)